### DISCUSSION TITLE
Let's update logo!

### DISCUSSION THREAD

**Discussion by pingwin840** [linuxmint/discussions#371] [Ideas]
The Linux Mint logo has a 2010's feel to it. It's high time for a major upgrade! A new logo has been designed by ARYLUNEIX, and I've attached it here. Let's modernize Linux Mint!
https://github.com/linuxmint/mint-themes/issues/471

**Comment by moloch1994** [comment]
I don't think this logo represents Linux Mint's overall style though. Linux Mint is not a modern bleeding-edge distro. It neither looks nor functions like one. In fact, I feel like half of its appeal is that it provides a familiar (in other words "old") desktop experience, opposing some of the more questionable innovations. This logo has a very futuristic font and a gradient - elements which you won't find anywhere in the actual system. And, while I would prefer for Mint to adopt more visual flair like gradients, transparency and blur, until then I think the current logo fits well with the current design.

**Reply by ghost** [reply]
I totally agree. Personally though, I just really like 2000s/2010s logos. They had so much life in them!
